### PR TITLE
Test review: Update collection.md

### PR DESCRIPTION
**Pull request by mglukhovsky**
Fixes an example collection API issue

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/rethinkdb/horizon-docs/pull/65" target="_blank">ReviewNinja

File: api/collection.md [hunk 1]
```diff
@@ -285,6 +285,18 @@ messages.find({id: 101});
 messages.find(101);
 ```
 
+You can use the Observaddsa `defaultIfEmpty` function to handle non-existing keys:
+```js
+var messages = hz('messages');
+messages.find(id).fetch().defaultIfEmpty().subscribe((msg) => {
+  if (msg == null) {
+    console.log("Message not found");
+    reburn;
```
Comment (mglukhovsky) [inline api/collection.md]: !fix Typo reburn -> return

File: api/collection.md [hunk 2]
```diff
@@ -285,6 +285,18 @@ messages.find({id: 101});
 messages.find(101);
 ```
 
+You can use the Observaddsa `defaultIfEmpty` function to handle non-existing keys:
+```py
```
Comment (mglukhovsky) [inline api/collection.md]: Why would you change this to Python?? !fix

File: api/collection.md (hunk 1 above)
Comment (mglukhovsky) [inline api/collection.md]: !fixed

**Comment by mglukhovsky** [comment]
!fix this could be a better pull request.

File: api/collection.md [hunk 3]
```diff
@@ -285,6 +285,18 @@ messages.find({id: 101});
...
+var messages = hz('messages');
+messages.find(id).fetch().defaultIfEmpty().subscribe((msg) => {
```
Comment (mglukhovsky) [inline api/collection.md]: Just commenting

**Comment by danielmewes** [comment]
For other readers (@chipotle): This is just a test PR for checking out ReviewNinja.

File: api/collection.md [hunk 4]
```diff
@@ -285,6 +285,18 @@ messages.find({id: 101});
 messages.find(101);
 ```
 
+You can use the Observaddsa `defaultIfEmpty` function to handle non-existing keys:
```
Comment (mglukhovsky) [inline api/collection.md]: This should be "Observables", you misspelled it! !fix

Comment (mglukhovsky) [inline api/collection.md]: !resolved

Comment (mglukhovsky) [inline api/collection.md]: This wasn't resolved!

Comment (mglukhovsky) [inline api/collection.md]: !fix Fix it!

**Pull request closed without merging**